spec/ios_extract_keys_from_strings_files_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'tmpdir'
+
+describe Fastlane::Actions::IosExtractKeysFromStringsFilesAction do
+  let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'translations', 'ios_extract_keys_from_strings_files') }
+
+  it 'extracts the right keys from Localizable.strings to InfoPlist.strings for all locales but the base one' do
+    Dir.mktmpdir('a8c-release-toolkit-tests-') do |tmp_dir|
+      # Arrange
+      lproj_source_dir = File.join(tmp_dir, 'Resources')
+      FileUtils.cp_r(File.join(test_data_dir, 'source/'), lproj_source_dir) # slash at end of `source/` makes sure we copy the dir content, not the dir itself
```

Comment:
What do you think of this?

```suggestion
      FileUtils.cp_r(File.join(test_data_dir, 'source', '.'), lproj_source_dir)
```